.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,40 @@
+{
+  "configurations": [
+    {
+      "name": "Linux",
+      "includePath": [
+        "${workspaceFolder}/code/include/**"
+      ],
+      "defines": [],
+      "compilerPath": "/usr/bin/gcc",

```

Comment:
Do you need to provide the absolute path here? Asking as it might not be the same for all distros out there.

Reply:
That's a good point, and I suspect it does not need to be an absolute path. vscode recommends absolute paths, but `/usr/bin/` would be in the `$PATH` environment for distros which use it, and `gcc` is likely to be in the path regardless if it's available. And on Windows it seems to work without the path prefix.

I'll push an update removing the `/usr/bin/` from this.

Reply:
One of my NAS has its compiler lying in `/opt/bin`, not that I plan on building orx on it though. :)

Reply:
https://github.com/orx/orx/pull/104/commits/3b8d6d54c33fe477230ab62464e69f4a46fc1112 should fix it!